### ISSUE TITLE
Hillshading and Contours issue.

### ISSUE THREAD

**Issue by carlodenardin** [openmaptiles/klokantech-terrain-gl-style#11]
For my own server I tried to add the style "Terrain" provided as open-source. But once I open the map with the correct style I'm not able to see the style in a properly way. It is flat. Maybe because hill shading and contours are not present in my mbtiles file.

Here I reported my style.json. I should change the url with something local.

```
"openmaptiles": {
      "type": "vector",
       "url": "mbtiles://{v3}"
    },
    "hillshading": {
      "type": "raster",
      "url": "https://klokantech.tilehosting.com/data/hillshades.json?key=tXme5cuqgrCqdPoZHqyn",
      "tileSize": 256
    },
    "contours": {
      "type": "vector",
      "url": "https://klokantech.tilehosting.com/data/contours.json?key=tXme5cuqgrCqdPoZHqyn"
    }
  },
  "glyphs": "{fontstack}/{range}.pbf"
}
```

<img width="515" alt="Screenshot 2022-02-04 at 00 46 02" src="https://user-images.githubusercontent.com/64202858/152448152-7ed40c60-b4ad-4eec-9594-4b2b60e4ff06.png">

How can we solve this issue? We should import the hill shading and contours from the mbtiles right?

**Comment by TomPohys** [comment]
Yes, for your server, you need to add mbtiles for hillshade and mbtiles for contours. If you add your API key from MapTiler Cloud (please see the current version of `style.json`) you can use the hillshade and contours from MapTiler. 

Every release has an asset (e.g. `v1.8.zip`) that includes a version for local hosing (`style-local.json`).

**Comment by wmgco** [comment]
I met a problem on JOSM.

<img width="312" height="350" alt="Image" src="https://github.com/user-attachments/assets/70b763d9-cd6a-40f8-8327-52c24a0fb562" />

<img width="586" height="393" alt="Image" src="https://github.com/user-attachments/assets/c43bc7be-86b2-430b-ad4b-2ad5a9af9875" />

the information of the error:

```
org.openstreetmap.josm.gui.mappaint.mapcss.parsergen.ParseException: Encountered " "}" "} "" at line 10, column 221.
Was expecting one of:
    <EOF> 
    "and" ...
    "or" ...
    "not" ...
    "@supports" ...
    <PP_NEWLINECHAR> ...
    <PP_WHITESPACE> ...
    "/*" ...
    <IDENT> ...
    <UINT> ...
    <STRING> ...
    <REGEX> ...
    "{" ...
    "(" ...
    ")" ...
    "," ...
    ":" ...
    <PP_SOMETHING_ELSE> ...
```
    
How can I solve it?
